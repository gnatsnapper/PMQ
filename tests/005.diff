001+ double free or corruption (out)
002+ Aborted (core dumped)
003+ 
004+ Termsig=6
001- 1
002- 1
003- 1
004- 1