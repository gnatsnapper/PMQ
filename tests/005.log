
---- EXPECTED OUTPUT
1
1
1
1
---- ACTUAL OUTPUT
double free or corruption (out)
Aborted (core dumped)

Termsig=6
---- FAILED
