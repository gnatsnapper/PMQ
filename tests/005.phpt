--TEST--
O_EXCL Flag Test
--SKIPIF--
<?php
if (!extension_loaded('pmq')) {
	 print 'skip';
}
?>
--FILE--
<?php
$name = '/testqueue' . bin2hex( random_bytes(8) );
$pmq1 = new PosixMessageQueue($name, PMQ_CREAT|PMQ_WRONLY);
try { $pmq2 = new PosixMessageQueue($name, PMQ_EXCL|PMQ_CREAT|PMQ_RDONLY); }
catch (Exception $e)
    {
        var_dump( $e->getMessage() );
    }
var_dump( $pmq1->unlink() );
?>
--EXPECT--
string(11) "File exists"
bool(true)
