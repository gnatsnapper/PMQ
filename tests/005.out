double free or corruption (out)
Aborted (core dumped)

Termsig=6